hc-nested-list.html

```diff
@@ -57,6 +57,12 @@
     Polymer({
       /* global hc */
       is: 'hc-nested-list',
+      properties: {
+        openMultipleNodes: {
```

Comment:
* Move this property into the behavior where the code is located.
* Rename it to `disableAutoCollapse`
* Add unit tests for the change in logic